### ISSUE TITLE
flatpak mask - error: Unsupported key: masked

### ISSUE THREAD

**Issue by admiralzeech** [flatpak/flatpak#3876]
## Description of the problem

I'm trying to use flatpak mask on qbittorrent to prevent updates.  When I run the command:

`flatpak mask org.qbittorrent.qBittorrent`

I get a password prompt and then the error:

`error: Unsupported key: masked`

## Steps to reproduce
- Install fresh Kubuntu 20.04.1
- Install flatpak via apt install
- Add flathub as a repo
- flatpak install flathub org.qbittorrent.qBittorrent
- flatpak mask org.qbittorrent.qBittorrent

--> Error.

**Comment by mwleeds** [comment]
Thanks for the bug report. This bug has been fixed by 73644b00c9b332ebba6be4cc5701dfb0ff0bc395 which will be in the next release of Flatpak.

**Comment by madscientist** [comment]
For those using Ubuntu 20.04 and can't upgrade, a workaround is to run flatpak as root via sudo:
```
~$ flatpak mask --remove somapp
error: Unsupported key: masked

~$ sudo flatpak mask --remove someapp

~$ flatpak mask
No masked patterns
```
